Add document for che installation on eks (#2808)
codenvy/che-docs: modules/administration-guide/pages/installing-che-on-AWS-EKS.adoc

:_content-type: PROCEDURE
:navtitle: Installing {prod-short} on the {eks} cluster
:description: Installing {prod-short} on the {eks} cluster

> :keywords: installing-{prod-short}-on-the-eks-cluster

Reviewer: ```suggestion
:keywords: installation, eks cluster
```

:page-aliases:

[id="installing-{prod-short}-on-AWS-EKS.adoc"]

> = Installing {prod-short} on the {eks} cluster

Reviewer: ```suggestion
= Installing {prod-short} on an EKS cluster
```

> 

Reviewer: docs usually have a short introductory paragraph about the process they describe. can you add it, please?

examples: 

- https://eclipse.dev/che/docs/stable/administration-guide/installing-che-on-the-virtual-kubernetes-cluster/
- https://eclipse.dev/che/docs/stable/administration-guide/installing-che-on-microsoft-azure/
- https://eclipse.dev/che/docs/stable/administration-guide/installing-che-on-openshift-using-cli/

.Prerequisites

> * A Che domain name

Reviewer: ```suggestion
* A {prod-short} domain name
```

> * A certificate for the che domain name: private and public keys in PEM format

Reviewer: ```suggestion
* A certificate (private and public keys in PEM format)  for the {prod-short} domain name
```

> * An external OIDC provider, you need the following information from your provider: Issuer URL, Client ID, Client Secret

Reviewer: ```suggestion
* An external OpenId Connect (OIDC) provider
* An OIDC issuer URL
* An OICD client ID
* An OICD client secret
```

> .Install required components

Reviewer: ```suggestion
.Installing required components
```

. Install link:https://kubernetes.github.io/ingress-nginx/[Ingress-Nginx Controller]
+

> [source,shell]

Reviewer: please check the Red Hat Style Guide section about commands with root privileges: https://redhat-documentation.github.io/supplementary-style-guide/#commands-with-root-privileges

you'll need to edit out mentions of shell in this doc

----
kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v1.11.1/deploy/static/provider/aws/deploy.yaml
----

. Install the link:https://github.com/devfile/devworkspace-operator/tree/main[devworkspace-operator]
+
[source,shell]
----
kubectl create namespace devworkspace
kubectl apply -f https://raw.githubusercontent.com/devfile/devworkspace-operator/refs/heads/main/deploy/deployment/kubernetes/combined.yaml
----

. Install the link:https://github.com/eclipse-che/che-operator[eclipse-che-operator]
+
[source,shell]
----
kubectl apply -f https://raw.githubusercontent.com/eclipse-che/che-operator/refs/heads/main/deploy/deployment/kubernetes/combined.yaml
----

> .Integrate the external OIDC provider with the EKS cluster

Reviewer: ```suggestion
.Integrating the external OIDC provider with the EKS cluster
```

* Select your EKS cluster, and then select the `Access` tab.
* In the OIDC Identity Providers section, select Associate Identity Provider.

> * On the Associate OIDC Identity Provider page, enter or select the following options, and then select Associate.

Reviewer: what's the correct name of the page, please?  Associate OIDC Identity Provider or  Associate Identity Provider

[subs="+quotes,attributes"]
----
For Name, enter a unique name for the provider.
For Issuer URL, enter the URL for your provider
For Client ID, enter the OIDC identity provider's client ID 
For Username claim, enter `email`
----

> .Create secret for the seft sign certificate

Reviewer: ```suggestion
.Creating secret for the seft-sign certificate
```

. Create a `che-tls` secret:
+
[subs="+quotes,attributes"]
----
$ {orch-cli} create secret tls che-tls \
--key <key_file> \
--cert <cert_file> \
-n {prod-namespace}
----

. Add the required labels to the secret:
+
[subs="+quotes,attributes"]
----
$ {orch-cli} label secret che-tls app.kubernetes.io/part-of=che.eclipse.org -n {prod-namespace}
----

> .Deploy che cluster

Reviewer: ```suggestion
.Deploying {prod-short} cluster
```

> . Prepare `CheCluster` manifest file

Reviewer: ```suggestion
. Prepare the `CheCluster` manifest file
```
